.devcontainer/Dockerfile

```diff
@@ -1,5 +0,0 @@
-ARG VARIANT=3.8
-FROM mcr.microsoft.com/vscode/devcontainers/python:${VARIANT}
-
-RUN pip3 install poetry pre-commit
-RUN poetry config virtualenvs.in-project true
```

Comment:
can you restore these files? 😊

Reply:
oh I though these were untracked, ofc

Reply:
they should be back now, but believe i screwed up the blame now, the history is still intact though.

Reply:
that's fine 😊